packages/build/src/target/ImportRewriter.ts

```diff
@@ -11,15 +11,17 @@ const APPLICATION_MODULE_INDICATORS = ['.', '/', 'http:', 'https:'];
 
 export default class ImportRewriter
 {
+    readonly #resources: ResourceList;
     readonly #module: Module;
     readonly #segmentation: Segmentation;
     readonly #segment: Segment | undefined;
 
     readonly #parser = new Parser();
     readonly #fileHelper = new FileHelper();
 
-    constructor(module: Module, segmentation: Segmentation, segment?: Segment)
+    constructor(resources: ResourceList, module: Module, segmentation: Segmentation, segment?: Segment)
```

Comment:
Flushing my thoughts: the resources and segments are both layers atop of the modules, so shouldn't the module be (stay) the main 'thing'?

```suggestion
    constructor(module: Module, resources: ResourceList, segmentation: Segmentation, segment?: Segment)
```

And yes, this also applies to the related files above and below...

Reply:
I was doubting about this, and decided to put the resources first. They are globally defined and for me more associated to an application than a module.